Fullstack-Development/Contribute/Readme.md

```diff
@@ -24,3 +24,7 @@
 - [Zulip Open Source Project](https://github.com/zulip/zulip/)
 
 - [More…](https://github.com/topics/fullstack)
+
+## Full Stack Web Development Bootcamps
```

Comment:
This section doesn't fall under contributing but resources. Please add it to the full stack development discussion so we can include it in roadmaps

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix